src/app/+item-page/edit-item-page/item-delete/item-delete.component.html

```diff
@@ -0,0 +1,96 @@
+<div class="container">
```

Comment:
I think there is room for some improvement regarding this component. Like given the user the real vision of how metadata would look like if he chooses to delete data (which fields and values will be copied after delete the record).

Reply:
Hi Paulo, thanks for your suggestion, but as you pointed out above, it won't be for this PR.